ibc_data/all_contrasts.tsv

```diff
@@ -1,300 +1,300 @@
 task	contrast	pretty_name	left_label	right_label	visual_form_recognition	visual_form_discrimination	visual_color_discrimination	visual_orientation	visual_attention	visual_working_memory	visual_buffer	shape_recognition	visual_tool_recognition	visual_body_recognition	visual_place_recognition	visual_face_recognition	visual_arithmetic_processing	visual_pattern_recognition	visual_pseudo_word_recognition	visual_word_recognition	visual_string_recognition	visual_tracking	vertical_checkerboard	horizontal_checkerboard	object_categorization	object_recognition	object_detection	feature_comparison	face_perception	emotional_face_recognition	emotion_expression_identification	facial_trustworthiness_recognition	facial_attractiveness_recognition	gender_discrimination	numerosity	auditory_perception	auditory_attention	auditory_memory	auditory_arithmetic_processing	sounds_perception	voice_perception	auditory_sentence_recognition	listening	speech_perception	speech_processing	language_processing	language_comprehension	semantic_processing	semantic_categorization	syntactic_parsing	sentence_processing	story_comprehension	reading	string_maintenance	word_maintenance	tool_maintenance	body_maintenance	place_maintenance	face_maintenance	animacy_perception	animacy_decision	motion_detection	response_selection	response_execution	left_finger_response_execution	left_hand_response_execution	left_toe_response_execution	tongue_response_execution	right_finger_response_execution	right_hand_response_execution	right_toe_response_execution	updating	working_memory	relational_comparison	grasping	spatial_cognition	spatial_localization	spatial_memory	spatial_working_memory	hand_chirality_recognition	hand_side_recognition	reward_processing	reward_valuation	incentive_salience	food_cue_reactivity	punishment_processing	confidence_judgment	reasoning	recognition	recall 	memory_retrieval	episodic_memory	temporal_cognition	time_perception	time_orientation	present-time_orientation	past-time_orientation	temporal_distance	empathy	imagined_physical_pain	imagined_emotional_pain	theory-of-mind	self-reference_effect	inhibition
-archi_standard	left-right_button_press	Left vs right hand button press	right hand	left hand																																																											1	1	1																																							
```

Comment:
oops, this is an unrelated change !

Reply:
Please fix this by separating your commits into different branches

Reply:
Done